jenkins/internal-ci/generic/branch/cortx-cc.groovy

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env groovy
+// Please configure branch and os_version as string parameters in Jenkins configuration manually.
+pipeline {
+    agent {
+        node {
+            label "docker-${os_version}-node"
+        }
+    }
+
+    // triggers {
+    //     pollSCM '*/5 * * * *'
+    // }
+    
+    environment {
+        version = "2.0.0"
+        env = "dev"
+        component = "cortx-cc"
+        release_dir = "/mnt/bigstorage/releases/cortx"
+        release_tag = "last_successful_prod"
+        build_upload_dir = "$release_dir/components/github/$branch/$os_version/$env/$component"
+    }
+    
+    options {
+        timeout(time: 120, unit: 'MINUTES')
+        timestamps()
+        ansiColor('xterm') 
+        disableConcurrentBuilds()  
+    }
+
+    stages {
+        stage('Checkout cortx-cc') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                dir ('cortx-cc') {
+                    checkout([$class: 'GitSCM', branches: [[name: "${branch}"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CloneOption', depth: 0, noTags: false, reference: '', shallow: false, timeout: 15], [$class: 'SubmoduleOption', disableSubmodules: false, parentCredentials: true, recursiveSubmodules: true, reference: '', trackingSubmodules: false, timeout: 15]], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', refspec: '+refs/heads/main:refs/remotes/origin/main', url: "https://github.com/Seagate/cortx-cc"]]])
+                }
+            }
+        }
+    
+        stage('Install Dependencies') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+
+                sh label: 'Install cortx-cc pre-requisites', script: '''
+                    yum -y install python3 python3-devel facter yum-utils
+                    yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
+                    yum -y install consul-1.9.1
+                '''
+
+                sh label: 'Install cortx-prereq', script: '''
+                    CORTX_UTILS_REPO_OWNER=$(echo $CORTX_UTILS_URL | cut -d "/" -f4)
```

Comment:
`CORTX_UTILS_URL` - This parameter is not defined in https://github.com/gauravchaudhari02/cortx-re/blob/cortx-cc-custom-ci/jenkins/internal-ci/generic/branch/cortx-cc.groovy 

We can reuse - https://github.com/Seagate/cortx-re/blob/main/jenkins/internal-ci/generic/branch/hare.groovy#L74

Reply:
Done